ajax/libs/material-components-web/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "material-components-web",
+  "version": "0.11.1",
+  "filename": "material-components-web.min.js",
+  "description": "Modular and customizable Material Design UI components for the web",
+  "homepage": "https://material.io/components/",
+  "keywords": [
+    "material components",
+    "material design"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Addy Osmani <@addyosmani> (https://www.addyosmani.com/)",
```

Comment:
As @traviskaufman said, Addy doesn't work on material-components-web, "The Material Design Team" might be a better choice, let's wait @traviskaufman for update.